format/opentimestamps/testdata/hello-world.fqtest

```diff
@@ -0,0 +1,1177 @@
+$ fq -d ots dv hello-world.txt.ots
```

Comment:
Once in the probe group i think you can skip `-d` so that the probing is tested also